src/cluster/command/CommandContext.ts

```diff
@@ -103,9 +103,20 @@ export class CommandContext<TChannel extends KnownTextableChannel = KnownTextabl
 
         throw new Error('Cannot queryMember without a guild!');
     }
+    public async queryUser(options: SlimEntityFindQueryOptions): Promise<ChoiceQueryResult<User>>;
+    public async queryUser(this: GuildCommandContext, options: Omit<SlimEntityFindQueryOptions, 'guild'>): Promise<ChoiceQueryResult<User>>;
```

Comment:
what's the point in overwriting the this here?

Reply:
Its to add the overloads that are accepted. The second overload is for the case where there is a guild associated with the context, so you dont need to supply one.